link-grammar/dict-common/regex-morph.c

```diff
@@ -182,16 +193,12 @@ static bool reg_comp(Regex_node *rn)
 	                            options, &rc, &erroffset, NULL);
 	if (re->re_code != NULL)
 	{
+		call_once(&call_once_flag, alloc_match_data);
```

Comment:
`#if HAVE_THREADS_H` is needed for emscripten.

Reply:
Gahh. I'll add that right now.

Reply:
OK, I pushed directly to master.

BTW I reviewed the regular regex code, I see nothing wrong with it, so I don't see why it would be failing your tests.

Reply:
> so I don't see why it would be failing your tests.

It was my mistake - I forgot the default any/amy/ady dicts doesn't work fine with the POSIX regex library.